website/src/util/RoadmapItems.ts

```diff
@@ -0,0 +1,114 @@
+/*****************************************************************/
+/**                                                              */
+/** GUIDE: ["Title", "Description", "Source", "Link to source"] **/
+/**                                                              */
+/*****************************************************************/
+
+// Product opportunities on our radar
+export const products_on_radar = [
+  ["Automatic caller ID lookup"],
+  ["Call recording"],
```

Comment:
Yes, it would. But I was trying to keep it as simple and "dumb" as possible for non-programmers to easily move around the items and edit their contents without getting overwhelmed or making small syntax errors. What do you think? :)

```javascript
export const coming_soon = [
  [
    "Smart & Personalized Voicemail",
    "Read your voicemail and custom-greet your contacts",
    "Voxist",
    "https://www.voxist.com/",
  ],
  [
    "Web Filtering for Kids",
    "Filter out website that contain harmful content",
    "WG2",
  ],
  ["SMS Spam Alert", "Get an alert when you receive SMS spam", "WG2"],
```

VS

```javascript
export const coming_soon = [
  {
    title: "Smart & Personalized Voicemail",
    text: "Read your voicemail and custom-greet your contacts",
    source: "Voxist",
    link: "https://www.voxist.com/",
  },
  {
    title: "Web Filtering for Kids",
    text: "Filter out website that contain harmful content",
    source: "WG2",
  },
  { title: "SMS Spam Alert", text: "Get an alert when you receive SMS spam", source: "WG2"},
```

In your experience, the people who'll be editing this file, are they comfortable/experienced enough to safely make changes to the programmer-friendly object version? :)

Reply:
@torotimes: Which syntax do you prefer?

Reply:
Can I choose the second one without you guys going ballistic on me? I think it's nice to have those tags as a reminder of what each thing is.

Reply:
The second option would be my preference too. The "downside" is that the so called keys (`title`, `text`, `source`, `link`) must be exactly so, or it will not render.

Reply:
Let's do #2 then! 

Thanks for reviewing and supporting the process, @jorunfa 

Are we good to approve the PR?

Reply:
I made a few comments down below on things that maybe should be fixed. Nothing too major. And then there is changing the code from the conclusion of the current discussion. 

The current PR (Pull Request) renders nicely, so I'm ok with merging it as-is, but would like the comments to be addressed/fixed either before or soon after merging the PR (in a new PR).

Reply:
That makes sense!

@Prid13 wanna go ahead and do that?